project/Dependencies.scala

```diff
@@ -180,9 +185,14 @@ object Dependencies {
 
   val actorTestkitTyped = l ++= Seq(Provided.junit, Provided.scalatest.value)
 
-  val remote = l ++= Seq(netty, aeronDriver, aeronClient, Test.junit, Test.scalatest.value, Test.jimfs)
+  val remoteDependencies = Seq(netty, aeronDriver, aeronClient)
+  val remoteOptionalDependencies = remoteDependencies.map(_ % "optional")
```

Comment:
Or Optional for no string usage

Reply:
:+1: will leave it to remain consistent with the rest of the file but we should switch them all

Reply:
Once this merges I'll do https://github.com/akka/akka/issues/27040